python/tests/test_sklearn_vw.py

```diff
@@ -201,7 +201,7 @@ def test_shuffle_pd_Series(self):
         # dummy data in vw format
         X = pd.Series(['1 |Pet cat', '-1 |Pet dog', '1 |Pet cat', '1 |Pet cat'], name='catdog')
 
-        kfold = KFold(n_splits=3, random_state=314, shuffle=False)
+        kfold = KFold(n_splits=3, random_state=314, shuffle=True)
```

Comment:
We need to either set (or leave) random_state=None or set shuffle=True. Seeing the name of the test function, I decided to set shuffle=True